model/registry/system.yaml

```diff
@@ -0,0 +1,255 @@
+groups:
+  # General system attributes
+  - id: registry.system
+    prefix: system
+    type: attribute_group
+    brief: "Describes System metric attributes"
+    attributes:
+      - id: device
+        type: string
+        stability: experimental
+        brief: "The device identifier"
+        examples: ["(identifier)"]
+  # system.cpu.* attribute group
+  - id: registry.system.cpu
```

Comment:
you can spare these blocks and just define things under `registry.system` i.e. `cpu.state`, `memory.state` etc similar to what we have done in process or db

Reply:
let's keep this discussion in a single place https://github.com/open-telemetry/semantic-conventions/pull/872#discussion_r1549913202